DDG4/plugins/Geant4ScintillationPhysics.cpp

```diff
@@ -80,6 +84,15 @@ namespace dd4hep {
              yes_no(m_finiteRiseTime), yes_no(m_stackPhotons),
              yes_no(m_trackSecondariesFirst));
         G4Scintillation* process = new G4Scintillation(name());
+#if G4VERSION_NUMBER >= 1070
+        G4OpticalParameters* params = G4OpticalParameters::Instance();
+        params->SetScintVerboseLevel(m_verbosity);
+        params->SetScintFiniteRiseTime(m_finiteRiseTime);
+        params->SetScintStackPhotons(m_stackPhotons);
+        params->SetScintTrackSecondariesFirst(m_trackSecondariesFirst);
+        params->SetScintYieldFactor(m_scintillationYieldFactor);
+        params->SetScintExcitationRatio(m_scintillationExcitationRatio);
```

Comment:
Could you add scintByParticleType and scintTrackInfo parameters?
https://github.com/Geant4/geant4/blob/6399a014b611a4aa09ecba225418222dac261ca1/source/processes/electromagnetic/utils/src/G4OpticalParameters.cc#L110-L111

Reply:
Implemented in 8e07cb7f9aa3b1c67aa607ab6bd6dbf6a0bcdd7a.